_rules/document-title-is-descriptive.md

```diff
@@ -0,0 +1,216 @@
+---
+name: Document title is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the document title describes the topic or purpose of the document.
+
+success_criterion: 
+- 2.4.2 # Page Titled (A)
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Corbb O’Connor 
+
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to the [document title](https://www.w3.org/TR/html5/single-page.html#dom-tree-accessors) in a [document](https://www.w3.org/TR/dom/#concept-document) where the [document element](https://www.w3.org/TR/dom/#document-element) is an HTML `html` element **and** the `title` element contains [text nodes](https://www.w3.org/TR/dom/#text) that do not only consist of [Unicode separator characters](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153).
```

Comment:
It is implicit that "the `title` element" is the "document title". This should be made explicit to avoid ambiguity.

Reply:
Align with https://github.com/auto-wcag/auto-wcag/pull/440/files